src/test/java/edu/ucla/library/iiif/auth/handlers/AbstractHandlerIT.java

```diff
@@ -64,6 +66,27 @@ public abstract class AbstractHandlerIT {
      */
     protected static final String POST_ITEMS_PATH = "/items";
 
+    /**
+     * The name of the HTTP request header used by the reverse proxy to carry the client IP address.
+     */
+    protected static final String X_FORWARDED_FOR = "X-Forwarded-For";
+
+    /**
+     * The fictitious client IP address that we'll pretend a reverse proxy sent through.
+     */
+    protected static final String FORWARDED_CLIENT_IP = "1.1.1.1";
```

Comment:
Might want to use an IP from the private ranges (https://en.wikipedia.org/wiki/Private_network), since 1.1.1.1 is a real, well-known DNS IP (https://en.wikipedia.org/wiki/1.1.1.1) and that might be confusing?